service/models.py

```diff
@@ -109,6 +109,10 @@ def init_db(cls, app):
         db.create_all()  # make our sqlalchemy tables
 
     @classmethod
+    def all(cls):
+        """ Returns all of the wishlists in the database"""
+        return cls.query.all()
+      
     def find(cls, wishlist_id):
```

Comment:
@wjduand due to the merge conflict resolution, two tests seem to be failing. We need `@classmethod` here as well since `find` is an operation that is performed on the database, rather than one row. It seems to have been lost while resolving the merge conflict.

Reply:
Tests should be passing now.